### PR TITLE
fix(config) Fix build time issue using defaut config

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Local build time ~540 seconds. Consequent builds for some reason take much less with a random difference varying from 82 seconds to 110 seconds per build using default `yarn run build` cmd

UPD:
* local build time is unstable, subsequent builds take less time (down to 82 seconds from 540 seconds) for unknown reason, antwar needs more profiling, dropping this work to not waste time
* this branch seems to build 20-25% faster than others due to node.js version bump
* this branch renders contributors only clientside, will finally fix error event not firing for 50X/40X image responses from OC
* redirects fixed, antwar.config.js redirects dont work at all in latest antwar version

### CODE REVIEW

**Comment by montogeek** [comment]
@EugeneHlushko It is faster?

**Comment by EugeneHlushko** [comment]
@montogeek locally it is stable at 92-106 seconds on my macbook pro 2015, i also noticed that after rebase not all of the redirects were in place and config's alignment went off. just fixed. seems node 8 speed up travis by 20% but not cut in half, trying to think of smth else meanwhile

**Comment by EugeneHlushko** [comment]
@montogeek can you pull this one and try out locally?

File: webpack.config.js [hunk 1]
```diff
@@ -101,12 +101,15 @@ module.exports = (env) => ({
     }),
     new RedirectWebpackPlugin({
       redirects: {
+        'support': '/contribute/',
```
Comment (montogeek) [inline webpack.config.js]: Where these new redirects come from?

File: antwar.config.js [hunk 2]
```diff
@@ -16,77 +16,61 @@ module.exports = () => ({
       layout: () => require('./src/components/Page/Page.jsx').default,
       content: () => require.context('./loaders/page-loader!./src/content', false, /^\.\/.*\.md$/),
       index: () => require('./src/components/Splash/Splash.jsx').default,
-      redirects: {
-        'support': '/contribute',
```
Comment (EugeneHlushko) [inline antwar.config.js]: from here

Comment (montogeek) [reply antwar.config.js]: Right, completely miss them

**Comment by montogeek** [comment]
Just ran `yarn build`:
```
real	16m16.693s
user	65m29.186s
sys	7m44.943s
```

**Comment by EugeneHlushko** [comment]
Ok profiling it down shows that antwar diving into too many operations at once, limiting workers doesnt solve this. for example if you only build "/" and "concepts" pages it takes you 10 seconds, when you build all the pages the overhead is starting to show on my laptop and `maximumWorkers` doesnt solve this for me locally. Suggest to leave as is and focus on removing antwar (`rebuild` branch) @montogeek

**Comment by EugeneHlushko** [comment]
I will rebase this after `migrate` section merge there are conflicts now

**Comment by EugeneHlushko** [comment]
I suggest we merge this, anyone else you want to look at this @montogeek ?

**Comment by montogeek** [comment]
Thanks

**Pull request merged**